tests/testsuite/build.rs

```diff
@@ -5119,13 +5311,25 @@ fn rustc_wrapper_from_path() {
     p.cargo("build -v")
         .env("RUSTC_WRAPPER", "wannabe_sccache")
         .with_status(101)
-        .with_stderr_contains("[..]`wannabe_sccache rustc [..]")
+        .with_stderr_data(str![[r#"
+[ERROR] could not execute process `wannabe_sccache rustc -vV` (never executed)
+
+Caused by:
+  [..]
```

Comment:
Platform dependent error:  

* No such file or directory (os error 2)
* program not found